src/Compute/Compute/help/Get-AzHost.md

```diff
@@ -0,0 +1,176 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Compute.dll-Help.xml
+Module Name: Az.Compute
+online version: https://docs.microsoft.com/en-us/powershell/module/az.compute/get-azhost
+schema: 2.0.0
+---
+
+# Get-AzHost
+
+## SYNOPSIS
+Get or list hosts.
+
+## SYNTAX
+
+### DefaultParameter (Default)
+```
+Get-AzHost [-ResourceGroupName] <String> [-HostGroupName] <String> [[-Name] <String>] [-InstanceView]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdParameter
+```
+Get-AzHost [-ResourceId] <String> [-InstanceView] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet will get a host in a host group.
+This cmdlet also lists all hosts in a host group if a host name is not given.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzHost -ResourceGroupName $resourceGroupName -HostGroupName $hostGroupName -Name $hostName
+
+ResourceGroupName    : myrg01
+PlatformFaultDomain  : 1
+AutoReplaceOnFailure : True
+HostId               : 00000000-0000-0000-0000-000000000000
+VirtualMachines[0]   : 
+  Id                 : /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myrg01/providers/Microsoft.Compute/virtualMachines/myvm01
+VirtualMachines[1]   : 
+  Id                 : /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myrg01/providers/Microsoft.Compute/virtualMachines/myvm02
+ProvisioningTime     : 7/27/2019 3:22:59 AM
+ProvisioningState    : Succeeded
+Sku                  : 
+  Name               : ESv3-Type1
+Id                   : /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myrg01/providers/Microsoft.Compute/hostGroups/myhostgroup01/hosts/myhost01
+Name                 : myhost01
+Location             : eastus
+Tags                 : {"key1":"val2"}
+```
+
+This command returns a host.
+
+### Example 2
+```
+PS C:\> Get-AzHost -ResourceGroupName $resourceGroupName -HostGroupName $hostGroupName -Name $hostName -InstanceView
```

Comment:
@hyonholee would you mind adding the corresponding output for the examples 2 and 3?